### ISSUE TITLE
Tag 4.11.0 still not available on Docker hub

### ISSUE THREAD

**Issue by jurgenhaas** [matomo-org/docker#293]
I see the images being updated several times in the last 24 hours, but the 4.11.0 tag is still not available on Docker hub. How long does it usually take before it's being available there? Just asking because we only recently switch to Docker based hosting and usually update quickly when new versions get announced. But now we seem to be stuck in a waiting loop ;-)

**Comment by jurgenhaas** [comment]
Hello, is this not maintained anymore?

**Comment by jurgenhaas** [comment]
Looks like now the tag is available, but the image still contains 4.10 unfortunately.

**Comment by benurb** [comment]
No, it doesn't.

```
docker run -it --rm matomo:latest cat core/Version.php | grep VERSION

const VERSION='4.11.0';
```

If you have defined /var/www/html as volume, it will not be automated automatically.

**Comment by jurgenhaas** [comment]
> If you have defined /var/www/html as volume, it will not be automated automatically.

OK, so how do you recommend the update process should look like? I thought, a force-recreate on the container should do, but obviously not.

**Comment by ontheair81** [comment]
I agree this is not very straight forward and looks pretty unusual to me, but this issue has been [covered](https://github.com/matomo-org/docker/issues/281) [several](https://github.com/matomo-org/docker/issues/161) [times](https://github.com/matomo-org/docker/issues/161) already.

In short:
- Update the container to a new version
- Trigger the web based update from within your installation

Let's see if this will change in the future. But for the time being, this is the upgrade process and always worked without any problems for me.

**Comment by jurgenhaas** [comment]
Well, all my deployments are fully automatic, so I have to find another way. What I've just tried was to shut down the app container, delete the directory on the host that was mounted into the container before, and then fire up the container again, that will recreate the app directory and it seems to work. Of course, config and log directory are located elsewhere and mounted into the container separately.
